charts/radar-jdbc-connector/templates/deployment.yaml

```diff
@@ -98,7 +98,7 @@ spec:
               command:
               - /bin/sh
               - -c
-              - curl -sf localhost:8083/connectors/radar-timescale-sink/status | grep -o '\"state\":\"[^\"]*\"' | tr '\\n' ',' | grep -vq FAILED || exit 1
+              - ((curl -sf localhost:8083/connectors/radar-timescale-sink/tasks/0/status | grep -o '\"state\":\"[^\"]*\"' | tr '\\n' ',' | grep -vq FAILED) && (curl -sf localhost:8083/connectors/radar-timescale-sink/tasks/1/status | grep -o '\"state\":\"[^\"]*\"' | tr '\\n' ',' | grep -vq FAILED)) || exit 1
```

Comment:
@blootsvoets Would it be ok to check the tasks' status here instead of the connector's status? I think the issue here was  the tasks' status was `FAILED` but the connector's status was still `RUNNING` when it loses the connection to timescaledb.

Reply:
@mpgxvii Should this check be also on `readinessProbe` or only `livenessProbe`?